fedbiomed/common/message.py

```diff
@@ -116,29 +132,61 @@ def __validate(self, fields: Dict[str, Any]) -> bool:
         for field_name, field_def in fields:
             value = getattr(self, field_name)
             if not isinstance(value, field_def.type):
-                logger.critical(f"{field_name}: '{value}' instead of '{field_def.type}'")
+                logger.critical(
+                    f"{field_name}: '{value}' instead of '{field_def.type}'"
+                )
                 ret = False
         return ret
 
+    def serialize(self) -> Dict:
```

Comment:
Sorry for discussing details, but the names `(de)serialize` look to me misleading. We are not doing serialization there, but conversion to/from `dict` to `Message`

Maybe rename as `to_dict` and `to_message` ? or other ....

Reply:
I leave you the last word to decide on that point ...